package/index.ts

```diff
@@ -4572,6 +4582,29 @@ export function runTransition(transition: IRouteTransition): void {
     nextIteration();
 }
 
+interface IBobrilAnchorCtx { lastHandledTransition: ITransitionRunInfo; lastHandledTransitionRunCount: number; }
+
+export function anchor(node: IBobrilNode): IBobrilNode {
+    postEnhance(node, {
+        postUpdateDom(ctx: IBobrilAnchorCtx, me: IBobrilNode, element: HTMLElement) {
+            if (!me.attrs || !lastRunnedTransitionInfo.transition)
+                return;
+
+            if (lastRunnedTransitionInfo.transition.name !== me.attrs.id)
```

Comment:
Using b.isActive would make more similar to b.link which could be beneficial. Why not support anchors with route parameters too?